uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -309,7 +309,7 @@ <h4 class="modal-title">
         </div>
       </div>
       <div
-        ng-show="($ctrl.isRole('manager') && $ctrl.statusBeforeEdit.value === '3') || $ctrl.directiveOptions.forStaff"
+        ng-show="$ctrl.isRole('manager') || $ctrl.directiveOptions.forStaff"
```

Comment:
ps refer to description of ticket as manager can edit and respond to most of the statuses and not just 3